src/main/resources/application.yml

```diff
@@ -23,6 +23,9 @@ ipfs:
   host: ${IEXEC_IPFS_HOST:127.0.0.1}
   port: ${IEXEC_IPFS_PORT:5001}
 
+jwt:
+  key-path: /data/.key
```

Comment:
```suggestion
  key-path: /data/jwt.key
```

Reply:
I went for a dot file so that they won't be displayed easily when exploring the file system.
Can we discuss your suggestion ?

Reply:
I understand you did that, but there is not much added value in pseudo hiding it on an admin host

Reply:
done in b91132d